### PR TITLE
Update finder to use new injection api.

### PR DESCRIPTION
**Pull request by seuros**

### CODE REVIEW

File: bin/console [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "bundler/setup"
+require "trailblazer/finder"
+
+# You can add fixtures and/or initialization code here to make experimenting
```
Comment (apotonick) [inline bin/console:7]: Do we need those files?

**Review by apotonick — APPROVED** [review]
Could we switch over to `assert_equal`? I don't see why we'd use RSpec here?

File: bin/console (hunk 1 above)
Comment (apotonick) [reply bin/console:7]: Also, if you wait till the new `dsl` version 1.0 is out we could use `Inject() => [...]` instead of the not-extendable `:inject`.

Comment (seuros) [reply bin/console:7]: We are still in version 0.x we can always change.

**Pull request merged**